README.md

```diff
@@ -31,4 +54,27 @@ console.log(v1.minus(v2));
 var v3 = new Vector(3, 4)
 console.log(v3.getLength());
 // => 5
-```
+
+class Vector {
+  constructor(x, y) {
+    this.x = x;
+    this.y = y;
+  }
+};
+
+Vector.prototype.plus = function (vector) {
+  let newVector = new Vector(this.x += vector.x, this.y += vector.y);
+  return newVector;
+};
+
+Vector.prototype.minus = function (vector) {
+  let newVector = new Vector(this.x -= vector.x, this.y -= vector.y);
+  return newVector;
+};
+
+Vector.prototype.getLength = function () {
+  let xySum = this.x * this.x + this.y * this.y;
+  let length = Math.sqrt(xySum);
+  return length;
+};
+````
```

Comment:
Gold star for the clear argument and variable names. Practice ES6 methods for writing classes.